ivy_tests/test_ivy/test_frontends/test_numpy/test_random/test_functions.py

```diff
@@ -136,6 +136,30 @@ def test_numpy_normal(
     )
 
 
+# shuffle
+@handle_frontend_test(
+    fn_tree="numpy.random.shuffle",
+    dtype_and_input=_get_dtype_and_matrix(),
```

Comment:
We shouldn't be testing only for matrix shaped arrays, we should use `helpers.dtype_and_values()` strategy here to be more general.